.travis.yml

```diff
@@ -44,5 +128,5 @@ deploy:
 notifications:
   email:
     recipients:
-    - mcgibbon@uw.edu
+#    - mcgibbon@uw.edu
```

Comment:
I want notifications :P

Reply:
absolutely!! I added this at a time when I was committing almost 10 times an hour and continually failing. Sorry about that!